src/sentry/lang/dart/utils.py

```diff
@@ -0,0 +1,110 @@
+import os
+import re
+
+import sentry_sdk
+
+from sentry.eventstore.models import Event
+from sentry.lang.java.utils import deobfuscation_template
+from sentry.models import Project, ProjectDebugFile
+from sentry.utils import json
+from sentry.utils.safe import get_path
+
+VIEW_HIERARCHY_TYPE_REGEX = re.compile(r"([^<>]+)(?:<([^<>]+)>)?")
```

Comment:
can we put a comment here explaining this regex?

Reply:
So I actually had a comment that I forgot to pull up here. Thanks for calling that out

https://github.com/getsentry/sentry/pull/46129/commits/2d2ac8ce75b453c9737b9c4fc79646947811016c